exla/lib/exla/mlir/context_pool.ex

```diff
@@ -0,0 +1,37 @@
+defmodule EXLA.MLIR.ContextPool do
+  @doc false
```

Comment:
```suggestion
  @moduledoc false
```

Reply:
Btw, let's `@moduledoc false` all MLIR modules, cause their APIs will certainly change once we remove xla mode.